proect_on_desktop.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <title>KMarketer</title>
+    <link rel="stylesheet" href="proect_on_desktop.css">
+</head>
+<body>
+    <section class="hero">
+        <div class="container">
+            <header class="header">
+                <div class="header_logo">
+                    <span>KMarketer</span>
+                </div>
+
+                <nav class="header_nav">
+                    <ul>
+                        <li class="nav_item">
+                            <a href="#" class="nav_link">Home</a>
+                        </li>
+                        <li class="nav_item">
+                            <a href="#" class="nav_link">Product</a>
+                        </li>
+                        <li class="nav_item">
+                            <a href="#" class="nav_link">Pricing</a>
+                        </li>
+                        <li class="nav_item">
+                            <a href="#" class="nav_link">Contact</a>
+                        </li>
+                    </ul>
+                </nav>
+
+                <div class="header_actions">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="img/search.svg" alt="Search"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/cart.svg" alt="Cart"  class="a_little_down_cart"></a>
+                        </li>
+                    </ul>
+                </div>
+            </header>
+
+            <div class="hero_title">
+                <div class="hero_content">
+                    <h1>We Take Care of <br> the Future</h1>
+                    <h4>We know how large objects will act, but things on a <br> small scale just do not act that way.</h4>
+                </div>
+
+                <div class="hero_actions">
+                    <button type="button" class="hero_button1">Get Quote Now</button>
```

Comment:
Better give these buttons 2 classes: 
boh will have class ```hero_button``` and style this class to match transparent button. 
Then you can add modifier to orange one for example make it ```.button--primary``` 
So all orange buttons will have modifier ```--primary```
I caould give you more details about buttons naming if you ask me today on the lessson. As it is really important to give correct and usefull class names

Reply:
**ok boss)**
I'll remind you at today's lecture